<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>560</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrueCrypt AutoMount</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/disc.png</normaloff>icons/disc.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>780</width>
     <height>519</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupItemList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>310</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>2</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>4</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTreeWidget" name="treeVolumeList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>290</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Drive</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Caption</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Used</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Free</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Serial Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Batch</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupControls">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>760</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="buttonMount">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Mount</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonMountAll">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Mount All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDismountAll">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Dismount All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonExit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>140</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDismount">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Dismount</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title">
      <string>Volume</string>
     </property>
     <widget class="QComboBox" name="comboVolumes">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>351</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>80</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Device ID:
Drive Name:
Drive Size:
Serial Number:
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelDeviceInfos">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelLogo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonPostMountBatch">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Post-Mount Batch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonUnassign">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Unassign</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAssign">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
    </widget>
    <zorder>groupVolume</zorder>
    <zorder>buttonMount</zorder>
    <zorder>buttonMountAll</zorder>
    <zorder>buttonDismountAll</zorder>
    <zorder>buttonExit</zorder>
    <zorder>buttonDismount</zorder>
    <zorder>buttonPostMountBatch</zorder>
    <zorder>buttonUnassign</zorder>
    <zorder>buttonAssign</zorder>
   </widget>
   <zorder>groupControls</zorder>
   <zorder>groupItemList</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVolumes">
    <property name="title">
     <string>Volumes</string>
    </property>
    <addaction name="actionSelect_File"/>
    <addaction name="actionSelect_Device"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionStart_with_Windows"/>
    <addaction name="actionStart_Minimized"/>
    <addaction name="actionAuto_Mount_on_Application_Start"/>
    <addaction name="actionAuto_Mount_on_Drive_Connect"/>
    <addaction name="separator"/>
    <addaction name="actionDefault_Password"/>
    <addaction name="actionPath_to_TrueCrypt"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Default_Keyfile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuVolumes"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionSelect_File">
   <property name="text">
    <string>Select File</string>
   </property>
  </action>
  <action name="actionSelect_Device">
   <property name="text">
    <string>Select Device</string>
   </property>
  </action>
  <action name="actionStart_with_Windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start with Windows</string>
   </property>
  </action>
  <action name="actionAuto_Mount_on_Application_Start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Mount on Application Start</string>
   </property>
  </action>
  <action name="actionAdd_Default_Keyfile">
   <property name="text">
    <string>Add Default Keyfile</string>
   </property>
  </action>
  <action name="actionPath_to_TrueCrypt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Path to TrueCrypt</string>
   </property>
  </action>
  <action name="actionDefault_Password">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default Password</string>
   </property>
  </action>
  <action name="actionAuto_Mount_on_Drive_Connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Mount on Drive Connect</string>
   </property>
  </action>
  <action name="actionStart_Minimized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Minimized</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>treeVolumeList</tabstop>
  <tabstop>comboVolumes</tabstop>
  <tabstop>buttonAssign</tabstop>
  <tabstop>buttonUnassign</tabstop>
  <tabstop>buttonMount</tabstop>
  <tabstop>buttonDismount</tabstop>
  <tabstop>buttonMountAll</tabstop>
  <tabstop>buttonDismountAll</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
